install.yaml

```
@@ -20,8 +20,8 @@ pre_install_actions:
   # You can also check for client DDEV version with ddev_version_constraint (see below).
   # - |
   #   #ddev-description:Checking DDEV version
-  #   if ! ( ddev debug capabilities 2>/dev/null | grep corepack >/dev/null 2>&1 ) ; then
-  #     echo "This add-on requires DDEV v1.23+ or higher, please upgrade." && exit 2
+  #   if ! ${DDEV_EXECUTABLE} debug match-constraint ">= 1.24.5" 2>/dev/null ; then
```

Add a default value:

```suggestion
  #   if ! ${DDEV_EXECUTABLE:-ddev} debug match-constraint ">= 1.24.5" 2>/dev/null ; then
```

---

I didn't know that there was such a variable. Ignore my suggestion.